goovn/apis.go

```diff
@@ -57,6 +57,13 @@ type OVNDBApi interface {
 	ASAdd(name string, addrs []string, external_ids map[string]string) *OvnCommand
 	// Delete addressset
 	ASDel(name string) *OvnCommand
+	// Add LB
+	LBAdd(name string, vipPort string, protocol string, addrs []string) *OvnCommand
+	// Delete LB with given name
+	LBDel(name string) *OvnCommand
+	// Update existing LB
+	LBUpdate(name string, vipPort string, protocol string, addrs []string) *OvnCommand
```

Comment:
Is it better to add LBUpdateMembers() to support more efficient updates when the size of members are big? This interface can support passing added and deleted members only, without re-providing unchanged members. (Of course, it doesn't have to be in this patch. It can be added later)

Reply:
Good suggestion. I will handle it in a follow up patch